src/tags/tests/test_views.py

```diff
@@ -207,6 +207,17 @@ def test_CreateView(self):
 
         self.assertTrue(Tag.objects.filter(name=form_data['name']).exists())
 
+    def test_CreateView__slug_validation(self):
+        """Invalid names provided to Tag CreateView should be rejected"""
```

Comment:
Better to be more explicit, here or inline code comment WHY it's invalid (must meet slug_validator and link to it)

Reply:
code comment fix added!